packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerPerf.test.ts

```diff
@@ -233,108 +232,6 @@ export function testPerf() {
 			}
 		});
 
-		describe("Peer commit rebasing for peer with advancing (but not tip) seq ref#", () => {
```

Comment:
This test is impractical to maintain because it measures multiple branch rebases over multiple EditManager operations.
The tests above and below it are more specific and (taken together) offer the same coverage.